test/ODataEndToEnd/Microsoft.OData.Service.Sample.Tests/TrippinE2ETestCases.cs

```diff
@@ -235,7 +235,7 @@ public void UQProperty()
 
             this.TestClientContext.AddToPeople(person);
             this.TestClientContext.SaveChanges();
-            int personId = person.PersonId;
+            long? personId = person.PersonId;
```

Comment:
Not related to derived type, before we change ID to long to cover an issue ID is long does not work. And proxy file is not updated.
This time, I need to update proxy file to include derived type, the ID is changed.